books/free-programming-books-langs.md

```diff
@@ -1115,6 +1116,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Exploring ES6](http://exploringjs.com/es6/) - Dr. Axel Rauschmayer (HTML)
 * [Google JavaScript Style Guide](https://google.github.io/styleguide/javascriptguide.xml) - Aaron Whyte, Bob Jervis, Dan Pupius, Erik Arvidsson, Fritz Schneider, & Robby Walker (HTML)
 * [Human JavaScript](http://read.humanjavascript.com/ch01-introduction.html) - Henrik Joreteg (HTML)
+* [Introduction to Professional Web Development in JavaScript](https://education.launchcode.org/intro-to-professional-web-dev/) - Chris Bay, Jim Flores, Blake Mills, Sally Steuterman, Paul Matthews, Carly Langlois (The LaunchCode Foundation)
```

Comment:
More or less the same... It has some non-printable content like gif / animations. E.g:

https://education.launchcode.org/intro-to-professional-web-dev/chapters/angular-lsn3/studio.html

So.. you are right at first time.

Reply:
Resolved in https://github.com/EbookFoundation/free-programming-books/pull/6540/commits/106eb8583651375a54400e43261e4d2930b5d4df